sdk/formrecognizer/azure-ai-formrecognizer/tests/recordings/test_business_card_async.pyTestBusinessCardAsynctest_business_card_v2.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
if the test doesn't create a recording you shouldn't need the recorded_by_proxy decorator